.github/workflows/extension-release-published.yml

```diff
@@ -72,6 +69,11 @@ jobs:
                 "id": "liquibase",
                 "username": "liquibot",
                 "password": "${{ secrets.LIQUIBOT_PAT }}"
+              },
+              {
+                "id": "sonatype-nexus-staging",
+                "username": "${{ secrets.SONATYPE_USERNAME }}",
+                "password": "${{ secrets.SONATYPE_TOKEN }}"
```

Comment:
@jandroav should these be:
```
"username": "MAVEN_USERNAME",
"password": "MAVEN_PASSWORD"
```
since they get set as env vars later in the worfklow?